src/components/navbars/DashboardNavStudent.jsx

```diff
@@ -0,0 +1,38 @@
+import Home from '../../assets/icons/home.png';
```

Comment:
Are you able to use SVG files instead of PNG. I don't often work with PNG but SVG supports better scalability.

Reply:
I used these icons when I created the mockups in Figma. I asked the team if we would use the icon library or use the ones in the mockups, and the team decided to use the icons from the mockups. Should we change them?